docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,271 @@
+The project overview
+====================
+
+When opening the dashboard the project overview page will be the first page greeting you.
+It has several elements:
+
+.. figure:: /dashboard/images/img_1.png
+   :width: 100%
+   :align: center
+   :alt: The project overview
+
+   The project overview page
+
+.. figure:: /dashboard/images/img_2.png
+   :width: 100%
+   :align: center
+   :alt: The project overview
+
+   The project overview page
+
+Let's go over what some of these buttons do:
+
+1. The inmanta "home" button: takes you back to this page.
+2. List of created projects: Lists all existing projects.
+   Clicking on the name of a project will take you to its page.
+3. Project delete button:
+   Deletes the project and all the environments it contains.
+   This will delete the history and environments, but it will not purge the system of changes made or managed by inmanta.
+4. Report an issue:
+   If you run into any issues/bugs, this button will take you to github with the "new issue" page open for the inmanta repository.
+5. Environment navigation button:
+   Displays a list of projects and their environmentes.
+   Allows for navigate to any environment, by simply clicking it's name.
+6. Create new project button:
+   This will take you through the creation of a new project and the creation of it's first environment.
+7. Green checkmark:
```

Comment:
Maybe also mention that this will change into a red cross when the dashboard lost its connection to the server.

Reply:
done